src/scripts/dad-joke.js

```diff
@@ -0,0 +1,42 @@
+const { directMention } = require("@slack/bolt");
+const axios = require("axios");
+const { cache } = require("../utils");
+
+module.exports = (app) => {
+  app.message(directMention(), /dad joke/i, async ({ say }) => {
+    const jokes = await cache("dad jokes", 60, async () => {
+      try {
+        const { data } = await axios.get(
+          "https://fatherhood.gov/jsonapi/node/dad_jokes"
+        );
+
+        if (data && data.data) {
+          return data.data.map((joke) => ({
+            setup: joke.attributes.field_joke_opener,
+            punchline: joke.attributes.field_joke_response,
+          }));
+        }
+        return [];
```

Comment:
Do we have access to the null-safe operator? Could remove some nesting if we wanted:

```suggestion
        return data?.data?.map((joke) => ({
          setup: joke.attributes.field_joke_opener,
          punchline: joke.attributes.field_joke_response,
        })) || [];
```